### DISCUSSION TITLE
Addin / tools version question

### DISCUSSION THREAD

**Discussion by minoseah629** [cake-build/cake#3824] [Q&A] [answered]
Hello, 
I am curious if there was a way to specify an addin or tools nuget version to be at least a specific version number instead of specific version.  Just trying to see if there is accommodation for nuget packages that is updated frequently. 
Thanks

**Comment by devlead** [comment, marked as answer]
Ranges are supported for addins and tools, example here:
https://github.com/cake-build/cake/blob/8f5f69c347a365fc31caf5f0ff251c7541ee5952/tests/integration/Cake.NuGet/InProcessInstaller.cake#L109

**Reply by minoseah629** [reply]
I’ll try this
